### PR TITLE
Convert from create-react-app to vite (https://github.com/reactjs/reactjs.org/pull/5487)

### PR DESCRIPTION
**Pull request by maria-mcparland**
Lots of stupid commits here while I got Amplify tests to work with Vite......

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,14 +23,13 @@ npm-debug.log*
...
 
-/dist
-
+dist
 .env.*
 .env
 build
 certs
 archive.zip
 body.js
```
Comment (KyleJohnst) [inline .gitignore:33]: Couple empty lines here

File: app/client/src/hooks/__tests__/useTransactionGet.test.tsx [hunk 2]
```diff
@@ -1,179 +1,48 @@
...
-const API_PATH = '/api/tsapi/v2/transactions?relativeDateType=CURRENT_DAY&accountIds=000000010975001,000000011008182,000000010975514,000000010900009';
+
 const queryClient = new QueryClient({
-  logger: {
-    log: console.log,
-    warn: console.warn,
-    // ✅ no more errors on the console for tests
-    error: process.env.NODE_ENV === 'test' ? () => {} : console.error,
-  },
+  // logger: {
```
Comment (KyleJohnst) [inline app/client/src/hooks/__tests__/useTransactionGet.test.tsx:13]: Any reason for this commented out code.

File: app/client/src/mocks/handler.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import {  rest } from 'msw'
```
Comment (KyleJohnst) [inline app/client/src/mocks/handler.ts:1]: Not important but if you have a linter it didnt seem to catch the space here.

Comment (maria-mcparland) [reply app/client/src/mocks/handler.ts:1]: so strict

**Review by KyleJohnst — APPROVED** [review]
Looks good :)

**Pull request merged**